packages/internals/src/wasm.ts

```diff
@@ -7,3 +8,10 @@ const { dependencies } = require('../package.json')
 export { prismaFmt }
 // e.g. 4.3.0-18.a39215673171b87177b86233206a5d65f2558857
 export const prismaFmtVersion: string = dependencies['@prisma/prisma-fmt-wasm']
+
+/**
+ * Set up a global registry for Wasm panics.
+ * This allows us to retrieve the panic message from the Wasm panic hook,
+ * which is not possible otherwise.
+ */
+globalThis.WASM_PANIC_REGISTRY = new WasmPanicRegistry()
```

Comment:
I'm not very comfortable with this name as it's too generic and could collide with something with an incompatible API if end users or maintainers of another library take a similar approach in solving the same problem. How about `PRISMA_WASM_PANIC_REGISTRY`?

Since this would need to be changed on the Rust side as well, it's totally okay to do that in follow-up PRs if we want to merge this one ASAP to unbreak `main`, but it's important that we do it before the release.

Reply:
Agree, let's do this before the release, but after this PR.

Reply:
Follow-up PR for Rust: https://github.com/prisma/prisma-engines/pull/3722